packages/lib/src/components/Pix/Pix.ts

```diff
@@ -0,0 +1,17 @@
+import QRLoaderContainer from '../helpers/QRLoaderContainer';
+
+class PixElement extends QRLoaderContainer {
+    public static type = 'pix';
+
+    formatProps(props) {
+        return {
+            delay: 2000, // ms
+            countdownTime: 15, // min
+            copyBtn: true,
+            introduction: 'Abra o app com sua chave PIX cadastrada, escolha Pagar com Pix e escaneie o QR Code ou copie e cole o código',
```

Comment:
Do we need this here and in the translation file?

Reply:
Oops! Good catch!